src/main/java/dev/openfeature/sdk/NoOpTransactionContextPropagator.java

```diff
@@ -0,0 +1,24 @@
+package dev.openfeature.sdk;
+
+/**
+ * A {@link TransactionContextPropagator} that simply returns null.
+ */
+public class NoOpTransactionContextPropagator implements TransactionContextPropagator {
+
+    /**
+     * {@inheritDoc}
+     * @return null
+     */
+    @Override
+    public EvaluationContext getTransactionContext() {
+        return null;
```

Comment:
It might be best to avoid null. Could be set this an instant variable. 

```suggestion
        return new ImmutableContext();
```